samples/forseti/forseti_conf.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################################
+# gsuite
```

Comment:
The comments here should be organized for the categories we have in the yaml. Otherwise this organization for their reasons of existence is confusing IMHO.

1. `global`
1. `inventory`
1. `scanner`
1. `enforcer`
1. `notifier`